service/entityresolution/entityresolution.go

```diff
@@ -1,17 +1,22 @@
 package entityresolution
 
 import (
+	"time"
+
 	"github.com/go-viper/mapstructure/v2"
 	"github.com/opentdf/platform/protocol/go/entityresolution"
 	"github.com/opentdf/platform/protocol/go/entityresolution/entityresolutionconnect"
 	claims "github.com/opentdf/platform/service/entityresolution/claims"
 	keycloak "github.com/opentdf/platform/service/entityresolution/keycloak"
+	"github.com/opentdf/platform/service/pkg/cache"
 	"github.com/opentdf/platform/service/pkg/serviceregistry"
 	"go.opentelemetry.io/otel/trace"
 )
 
 type ERSConfig struct {
-	Mode string `mapstructure:"mode" json:"mode"`
+	Mode            string `mapstructure:"mode" json:"mode"`
+	CacheExpiration string `mapstructure:"cacheexpiration" json:"cacheexpiration"`
```

Comment:
@jrschumacher 
I think you can take a `time.Duration` here. Then a user would set like `30s` or `10m`.